test/lecture.js

```diff
@@ -448,7 +572,7 @@ describe("Cancel a booked lecture ", async function ()  {
       `/api/lectures/${lectureBookingTuple.lecture_id}/cancelbook`
     );
     expect(res).to.have.property("body");
-    expect(res.body.message).to.equal(`Booking canceled.`);
+    expect(res.body.message).to.equal(`Booking cancelled.`);
```

Comment:
Before this line, two delete methods are called. The first with the changes in the back-end should work, the second fail because after the deleting there's no more booking to cancel, so it should expect a message equals to the error of no booking to cancel